pkg/apis/kops/networking.go

```diff
@@ -248,12 +248,20 @@ type RomanaNetworkingSpec struct {
 
 // AmazonVPCNetworkingSpec declares that we want Amazon VPC CNI networking
 type AmazonVPCNetworkingSpec struct {
-	// The container image name to use
-	ImageName string `json:"imageName,omitempty"`
-	// The init container image name to use
-	InitImageName string `json:"initImageName,omitempty"`
+	// AwsVpcK8sCniConfigureRpFilter specifies whether ipamd should configure rp filter for primary interface.
+	// Setting this to false will require rp filter to be configured through init container.
+	AwsVpcK8sCniConfigureRpFilter string `json:"awsVpcK8sCniConfigureRpFilter,omitempty"`
 	// Env is a list of environment variables to set in the container.
 	Env []EnvVar `json:"env,omitempty"`
+	// DisableTCPEarlyDemux should be set to true for amazon-k8s-cni-init container under initcontainers if ENABLE_POD_ENI is set to true,
+	// in order for the kubelet to connect via TCP (for liveness or readiness probes) to pods that are using per pod security groups.
+	// This will increase the local TCP connection latency slightly.
+	// To use this setting, a Linux kernel version of at least 4.6 is needed on the worker node.
+	DisableTCPEarlyDemux string `json:"disableTCPEarlyDemux,omitempty"`
```

Comment:
Also a boolean.

We try to avoid negative booleans.

```suggestion
	TCPEarlyDemux *bool `json:"enableTCPEarlyDemux,omitempty"`
```